ui/src/modals/newtask.vue

```diff
@@ -264,35 +264,54 @@ export default {
             });
 
             //now find apps that user can submit
-            this.$http.get('app', {params: {
-                find: JSON.stringify({
-                    "inputs.datatype": {$in: datatype_ids},
-                    removed: false,
-                }),
+            this.$http.get('/app/query', {params: {
                 sort: 'name', 
                 populate: 'inputs.datatype outputs.datatype',
                 limit: 500, //TODO - this is not sustailable
+                incompatible: true
             }})
             .then(res=>{
                 //now, pick apps that we have *all* input datasets that matches the input datatype/tags
```

Comment:
is this comment still true?

Reply:
removed this logic from the ui moved to backend